app/views/listings/index.html.erb

```diff
@@ -1,2 +1,14 @@
-<h1>Listings#index</h1>
-<p>Find me in app/views/listings/index.html.erb</p>
+<h1><%= @listings.count %> items found</h1>
+
+<div class = "cards-listing">
+  <% @listings.each do |listing| %>
+    <div class = "card-listing">
+      <% if listing.photo.attached? %>
+        <%= cl_image_tag listing.photo.key, height:300, width:400, crop: :fill %>
```

Comment:
Because it's possible to update multiple photos, photos will be an array of one or more items. Think we need to have an 'If listing.photos.attached?' then do photos.first,key to show the first photo attached as the image